### PR TITLE
Add more detail for needing ambassador auth service in AES for external filters to work

### PR DESCRIPTION
**Pull request by edalford11**
## Description
I ran into an issue when upgrading to AES where the docs had said not to use AuthService and to use External Filters. Because of this, I had turned off the AuthService flag in the helm chart but I didn't realize that External Filters relies on this internal AuthService that ambassador provides. This makes that dependency more clear.

## Related Issues
#2831

### CODE REVIEW

File: docs/topics/running/services/auth-service.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ The Ambassador API Gateway provides a highly flexible mechanism for authenticati
...
 
-Because of the limitations described above, **the Ambassador Edge Stack does not support `AuthService` resources, and you should instead use an [`External` `Filter`](../../../using/filters/external),** which is mostly a drop-in replacement for an `AuthService`.
+Because of the limitations described above, **the Ambassador Edge Stack does not support `AuthService` resources, and you should instead use an [`External` `Filter`](../../../using/filters/external),** which is mostly a drop-in replacement for an `AuthService`. Note at AES deploys its own AuthService, which External Filter relies on. You must be sure the AES AuthService exists before External Filters will work.
```
Comment (richarddli) [inline docs/topics/running/services/auth-service.md:7]: Thanks! A grammar nit. How about:

The External Filter relies on the AES AuthService. Make sure the AES AuthService is deployed before configuring External filters.

**Review by richarddli — CHANGES_REQUESTED** [review]

File: docs/topics/running/services/auth-service.md (hunk 1 above)
Comment (edalford11) [reply docs/topics/running/services/auth-service.md]: Fixed!

**Comment by edalford11** [comment]
@richarddli does this look good to you?

**Comment by edalford11** [comment]
@richarddli bump

**Review by kflynn — APPROVED** [review]
Thanks much!

**Review by richarddli — APPROVED** [review]

**Pull request merged**